content/consulting/nextjs.mdx

```diff
@@ -0,0 +1,89 @@
+---
+type: consulting
+booking:
+  title: >-
+    Want to build a web app with <span
+    class="text-sswRed">Next.js</span>?
+  subTitle: 'Create performant and SEO-friendly web apps with Next.js'
+  videoBackground: /images/videos/MVC_background.mp4
+benefits:
+  benefitList:
+    - image: /images/benefits/look-into-the-future.png
+      title: SERVER-SIDE RENDERING
+      description: >
+        Next.js provides automatic static optimisation to render your React 
+        application on the server to static HTML.  
+    - image: /images/benefits/integration.png
+      title: SEO FRIENDLY
+      description: >
+        Compared to a client-side rendered single page application (SPA), Next.js
+        has much better SEO capabilities. Next.js can generate static HTML pages
+        that are easily indexable by search engines.
+    - image: /images/benefits/fast.png
+      title: API ROUTES
+      description: >
+        Next.js allows for the quick and easy creation of API routes via file-based routing to handle server-side
+        logic. This allows for the fast creation of a backend API without the need
+        for a separate backend server. 
+    - image: /images/benefits/look-into-the-future.png
+      title: EDGE RUNTIME
+      description: >
+        Next.js APIs can optionally be deployed to the edge, meaning that API functionality
+        can be run on a server close to the end-user, reducing latency. 
+    - image: /images/benefits/ux.png
+      title: REACT SERVER COMPONENTS
+      description: >
+        React Server Components means you can now run code with a Node.js runtime 
+        within your React components.
+    - image: /images/benefits/self-service.png
+      title: AUTOMATIC OPTIMISATION
+      description: >
+        Next.js features built-in automatic optimisation of images, fonts, and CSS, 
+        resulting in major performance improvements for users.
+  rule:
+    - name: SSW Rules to Better Next.js
+      url: 'https://www.ssw.com.au/rules/rules-to-better-nextjs/'
+technologies:
+  header: Related Technologies
+  technologyCards:
+    - technologyCard: content/technologies/nextJs.mdx
+    - technologyCard: content/technologies/node-js.mdx
+    - technologyCard: content/technologies/react.mdx
+solution:
+  project: Next.js
+callToAction: 'Talk to us about your {{TITLE}} project'
+seo:
+  title: Enterprise Development and Consulting for Next.js Apps
+---
+
+# Why build your web app with **Next.js**?
+
+Next.js is a React framework that provides a wide range useful features for your web app, 
+such as server-side rendering, automatic optimisation, and API routes. It has become one of the 
+industry-leading frameworks for building performant and scalable React-based web apps.
+
+<VideoEmbed url="https://www.youtube.com/watch?v=Sklc_fQBmcs" />
```

Comment:
Good video, but not SSW - I'd add a long term goal to your Trello board to make a cool NextJS video